commit 0fd838688ac2ac75fdc8a8db2e6f84fbfd249887
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:18 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
index 20d3002..7f00672 100644
--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -50,7 +50,7 @@ ecm_generate_headers(KPim${KF_MAJOR_VERSION}AkonadiSearchCore_CamelCasecore_HEAD
     PREFIX Core
     )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchCore EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchCore EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(FILES
     ${KPim${KF_MAJOR_VERSION}AkonadiSearchCore_CamelCasecore_HEADERS}
diff --git a/debug/CMakeLists.txt b/debug/CMakeLists.txt
index 80266ee..fbc3538 100644
--- a/debug/CMakeLists.txt
+++ b/debug/CMakeLists.txt
@@ -48,7 +48,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}AkonadiSearchDebug PROPERTIES
 
 target_include_directories(KPim${KF_MAJOR_VERSION}AkonadiSearchDebug INTERFACE "$<INSTALL_INTERFACE:${KDE_INSTALL_INCLUDEDIR}/KPim${KF_MAJOR_VERSION}/AkonadiSearch>")
 
-install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchDebug EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchDebug EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(FILES
     akonadisearchdebugdialog.h
diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index b1d812d..2157847 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -64,7 +64,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}AkonadiSearchPIM PROPERTIES
 
 target_include_directories(KPim${KF_MAJOR_VERSION}AkonadiSearchPIM INTERFACE "$<INSTALL_INTERFACE:${KDE_INSTALL_INCLUDEDIR}/KPim${KF_MAJOR_VERSION}/AkonadiSearch>")
 
-install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchPIM EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchPIM EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 set(AKONADISEARCH_INCLUDES
     query.h
diff --git a/xapian/CMakeLists.txt b/xapian/CMakeLists.txt
index 0a51028..d39cde5 100644
--- a/xapian/CMakeLists.txt
+++ b/xapian/CMakeLists.txt
@@ -44,7 +44,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}AkonadiSearchXapian PROPERTIES
 
 generate_export_header(KPim${KF_MAJOR_VERSION}AkonadiSearchXapian BASE_NAME AKONADI_SEARCH_XAPIAN EXPORT_FILE_NAME search_xapian_export.h)
 
-install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchXapian EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}AkonadiSearchXapian EXPORT KPim${KF_MAJOR_VERSION}AkonadiSearchTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(FILES
     xapiandatabase.h
