commit 7c49ddded79e574c364eba4fe116303ed17469eb
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:21 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/akonadi-contacts/CMakeLists.txt b/src/akonadi-contacts/CMakeLists.txt
index 560942c4..d974f515 100644
--- a/src/akonadi-contacts/CMakeLists.txt
+++ b/src/akonadi-contacts/CMakeLists.txt
@@ -233,7 +233,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}AkonadiContact PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}AkonadiContact
-    EXPORT KPim${KF_MAJOR_VERSION}AkonadiContactTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}AkonadiContactTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_pri_file(BASE_NAME AkonadiContact
diff --git a/src/contact-editor/CMakeLists.txt b/src/contact-editor/CMakeLists.txt
index db08f58b..0f5884a9 100644
--- a/src/contact-editor/CMakeLists.txt
+++ b/src/contact-editor/CMakeLists.txt
@@ -177,7 +177,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}ContactEditor PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}ContactEditor
-    EXPORT KPim${KF_MAJOR_VERSION}ContactEditorTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}ContactEditorTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_pri_file(BASE_NAME ContactEditor
