diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b589aa..4235c27 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,6 +12,7 @@ set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
 include(GenerateExportHeader)
 include(ECMGenerateHeaders)
 include(CMakePackageConfigHelpers)
+include(CMakeDependentOption)
 include(ECMSetupVersion)
 include(FeatureSummary)
 include(KDEInstallDirs)
@@ -22,6 +23,12 @@ include(ECMAddTests)
 include(ECMQtDeclareLoggingCategory)
 include(ECMCoverageOption)
 
+cmake_dependent_option(
+    AKONADISEARCH_ENABLE_KRUNNER_RUNNER
+    "Build KRunner runner to search contacts" ON
+    "NOT WIN32" OFF
+)
+
 set(AKONADISEARCH_VERSION ${PIM_VERSION})
 
 set(AKONADI_VERSION "5.9.0")
@@ -33,7 +40,7 @@ set(QT_REQUIRED_VERSION "5.9.0")
 
 find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Core Test)
 find_package(KF5 ${KF5_VERSION} REQUIRED COMPONENTS I18n Config Crash KCMUtils)
-if (NOT WIN32)
+if (AKONADISEARCH_ENABLE_KRUNNER_RUNNER)
     find_package(KF5Runner ${KF5_VERSION} REQUIRED)
 endif()
 
@@ -86,7 +93,7 @@ add_subdirectory(lib)
 add_subdirectory(akonadiplugin)
 add_subdirectory(search)
 add_subdirectory(debug)
-if (NOT WIN32)
+if (AKONADISEARCH_ENABLE_KRUNNER_RUNNER)
     add_subdirectory(runner)
 endif()
 
