diff --git a/configuration/akregator_config_plugins.cpp b/configuration/akregator_config_plugins.cpp
index e70ef3b0..384d4bbf 100644
--- a/configuration/akregator_config_plugins.cpp
+++ b/configuration/akregator_config_plugins.cpp
@@ -13,9 +13,15 @@
 #include <PimCommon/ConfigurePluginsListWidget>
 #include <PimCommon/ConfigurePluginsWidget>
 #include <QHBoxLayout>
-using namespace Akregator;
 
-K_PLUGIN_FACTORY(KCMAkregatorPluginsConfigFactory, registerPlugin<KCMAkregatorPluginsConfig>();)
+namespace Akregator {
+
+K_PLUGIN_FACTORY(KCMAkregatorPluginsConfigFactory, registerPlugin<Akregator::KCMAkregatorPluginsConfig>();)
+
+}
+
+
+using namespace Akregator;
 
 KCMAkregatorPluginsConfig::KCMAkregatorPluginsConfig(QWidget *parent, const QVariantList &args)
     : KCModule(parent, args)
