commit f87439ade6af6d3404036a0456dc444f8b6a6162
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:16 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 224475c..3bdf532 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -56,7 +56,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}GrantleeTheme PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}GrantleeTheme
-    EXPORT KPim${KF_MAJOR_VERSION}GrantleeThemeTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}GrantleeThemeTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
 )
 
 ecm_generate_headers(GrantleeTheme_CamelCase_HEADERS
