commit a9abddb9a3cdb738425232f439f159f7326961c7
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:17 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
index 353c9d63..84529b56 100644
--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -51,7 +51,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}IdentityManagement PROPERTIES
     EXPORT_NAME IdentityManagement
 )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}IdentityManagement EXPORT KPim${KF_MAJOR_VERSION}IdentityManagementTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}IdentityManagement EXPORT KPim${KF_MAJOR_VERSION}IdentityManagementTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### Generate Headers ###############
 ecm_generate_headers(KIdentityManagement_CamelCase_HEADERS
diff --git a/src/widgets/CMakeLists.txt b/src/widgets/CMakeLists.txt
index 7f245e5a..77463e54 100644
--- a/src/widgets/CMakeLists.txt
+++ b/src/widgets/CMakeLists.txt
@@ -44,7 +44,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}IdentityManagementWidgets PROPERTIE
     EXPORT_NAME IdentityManagementWidgets
 )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}IdentityManagementWidgets EXPORT KPim${KF_MAJOR_VERSION}IdentityManagementTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}IdentityManagementWidgets EXPORT KPim${KF_MAJOR_VERSION}IdentityManagementTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### Generate Headers ###############
 ecm_generate_headers(KIdentityManagementWidgets_CamelCase_HEADERS
