commit b9a309933b6351816e5e1b934c0de1781bf7bac6
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:22 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/kmailtransport/CMakeLists.txt b/src/kmailtransport/CMakeLists.txt
index 590d311..4c07dd3 100644
--- a/src/kmailtransport/CMakeLists.txt
+++ b/src/kmailtransport/CMakeLists.txt
@@ -98,7 +98,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MailTransport PROPERTIES
     )
 
 
-install(TARGETS KPim${KF_MAJOR_VERSION}MailTransport EXPORT KPim${KF_MAJOR_VERSION}MailTransportTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}MailTransport EXPORT KPim${KF_MAJOR_VERSION}MailTransportTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(FILES mailtransport.kcfg DESTINATION ${KDE_INSTALL_KCFGDIR})
 
diff --git a/src/kmailtransportakonadi/CMakeLists.txt b/src/kmailtransportakonadi/CMakeLists.txt
index b77301a..6731bc7 100644
--- a/src/kmailtransportakonadi/CMakeLists.txt
+++ b/src/kmailtransportakonadi/CMakeLists.txt
@@ -67,7 +67,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MailTransportAkonadi PROPERTIES
     )
 
 
-install(TARGETS KPim${KF_MAJOR_VERSION}MailTransportAkonadi EXPORT KPim${KF_MAJOR_VERSION}MailTransportAkonadiTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}MailTransportAkonadi EXPORT KPim${KF_MAJOR_VERSION}MailTransportAkonadiTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ecm_generate_headers(MailTransport_kmailtransportakonadi_CamelCase_HEADERS
     HEADER_NAMES
