commit fbe2b3f3a2feaa8f2b4109e3523f80e2f15c12f2
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:15 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index bf09ab6..d19222c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -42,7 +42,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}Mbox PROPERTIES
     EXPORT_NAME Mbox
 )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}Mbox EXPORT KPim${KF_MAJOR_VERSION}MboxTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}Mbox EXPORT KPim${KF_MAJOR_VERSION}MboxTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### Generate Headers ###############
 ecm_generate_headers(KMbox_CamelCase_HEADERS
