commit b1b87b61073e725e54caf33c1fe6093595999a2d
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:17 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 0b30191..001ea76 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -58,7 +58,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}KontactInterface PROPERTIES
 )
 
 
-install(TARGETS KPim${KF_MAJOR_VERSION}KontactInterface EXPORT KPim${KF_MAJOR_VERSION}KontactInterfaceTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}KontactInterface EXPORT KPim${KF_MAJOR_VERSION}KontactInterfaceTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### install files ###############
 
