commit dd87b222f2db976cdaf43e5319439e20f89c55d9
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:20 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 5cb2dbf..4c8d00e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -67,7 +67,7 @@ ecm_generate_headers(KSMTP_CamelCase_HEADERS
     REQUIRED_HEADERS KSMTP_HEADERS
 )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}SMTP EXPORT KPim${KF_MAJOR_VERSION}SMTPTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}SMTP EXPORT KPim${KF_MAJOR_VERSION}SMTPTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/ksmtp_export.h
