commit c915bbdf1541bc39005399ce72ec3a21c1b2cebc
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:16 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index da79665..af05226 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -62,7 +62,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}Gravatar PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}Gravatar
-    EXPORT KPim${KF_MAJOR_VERSION}GravatarTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}GravatarTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
 )
 
 ecm_generate_headers(Gravatar_Camelcase_job_HEADERS
