commit 67d24f77a634128905318f9830d0c8fb8321b6ed
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:16 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/messagecomposer/src/CMakeLists.txt b/messagecomposer/src/CMakeLists.txt
index a52cb8792..6898abad2 100644
--- a/messagecomposer/src/CMakeLists.txt
+++ b/messagecomposer/src/CMakeLists.txt
@@ -311,7 +311,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MessageComposer PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}MessageComposer
-    EXPORT KPim${KF_MAJOR_VERSION}MessageComposerTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}MessageComposerTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_headers(MessageComposer_Camelstatusbarwidget_HEADERS
diff --git a/messagecore/src/CMakeLists.txt b/messagecore/src/CMakeLists.txt
index ca1ab88ec..91fe3fc73 100644
--- a/messagecore/src/CMakeLists.txt
+++ b/messagecore/src/CMakeLists.txt
@@ -114,7 +114,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MessageCore PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}MessageCore
-    EXPORT KPim${KF_MAJOR_VERSION}MessageCoreTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}MessageCoreTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_headers(MessageCore_Camelcasemisc_HEADERS
diff --git a/messagelist/src/CMakeLists.txt b/messagelist/src/CMakeLists.txt
index 1802fb865..363fb31fa 100644
--- a/messagelist/src/CMakeLists.txt
+++ b/messagelist/src/CMakeLists.txt
@@ -160,7 +160,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MessageList PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}MessageList
-    EXPORT KPim${KF_MAJOR_VERSION}MessageListTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}MessageListTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_headers(MessageList_Camelcasecore_HEADERS
diff --git a/messageviewer/src/CMakeLists.txt b/messageviewer/src/CMakeLists.txt
index ed53fd594..ebe08f05f 100644
--- a/messageviewer/src/CMakeLists.txt
+++ b/messageviewer/src/CMakeLists.txt
@@ -464,7 +464,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}MessageViewer PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}MessageViewer
-    EXPORT KPim${KF_MAJOR_VERSION}MessageViewerTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}MessageViewerTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_headers(MessageViewer_Camelcasewebengine_HEADERS
diff --git a/mimetreeparser/src/CMakeLists.txt b/mimetreeparser/src/CMakeLists.txt
index d8c7834b6..7ceec8840 100644
--- a/mimetreeparser/src/CMakeLists.txt
+++ b/mimetreeparser/src/CMakeLists.txt
@@ -173,7 +173,7 @@ endif()
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}MimeTreeParser
-    EXPORT KPim${KF_MAJOR_VERSION}MimeTreeParserTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}MimeTreeParserTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 set_target_properties(KPim${KF_MAJOR_VERSION}MimeTreeParser PROPERTIES
diff --git a/webengineviewer/src/CMakeLists.txt b/webengineviewer/src/CMakeLists.txt
index e18056aa7..afb2d573b 100644
--- a/webengineviewer/src/CMakeLists.txt
+++ b/webengineviewer/src/CMakeLists.txt
@@ -162,7 +162,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}WebEngineViewer PROPERTIES
 
 install(TARGETS
     KPim${KF_MAJOR_VERSION}WebEngineViewer
-    EXPORT KPim${KF_MAJOR_VERSION}WebEngineViewerTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KPim${KF_MAJOR_VERSION}WebEngineViewerTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 ecm_generate_headers(WebEngineViewer_Camelfindbar_HEADERS
