commit f97f8fc3eb599564c9e33e53399f6db659bd4dd8
Author: Laurent Montel <montel@kde.org>
Date:   Sat Apr 22 09:27:20 2023 +0200

    Use correct INSTALL_TARGETS_DEFAULT_ARGS

diff --git a/src/pimcommon/CMakeLists.txt b/src/pimcommon/CMakeLists.txt
index 55193793..76891728 100644
--- a/src/pimcommon/CMakeLists.txt
+++ b/src/pimcommon/CMakeLists.txt
@@ -174,7 +174,7 @@ set_target_properties(KF5PimCommon PROPERTIES
 
 install(TARGETS
     KF5PimCommon
-    EXPORT KF5PimCommonTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KF5PimCommonTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 if (BUILD_TESTING)
diff --git a/src/pimcommonakonadi/CMakeLists.txt b/src/pimcommonakonadi/CMakeLists.txt
index 373c1317..3dd2ee25 100644
--- a/src/pimcommonakonadi/CMakeLists.txt
+++ b/src/pimcommonakonadi/CMakeLists.txt
@@ -189,7 +189,7 @@ set_target_properties(KF5PimCommonAkonadi PROPERTIES
 
 install(TARGETS
     KF5PimCommonAkonadi
-    EXPORT KF5PimCommonAkonadiTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS}
+    EXPORT KF5PimCommonAkonadiTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS}
     )
 
 if (BUILD_TESTING)
